app/controllers/devise/saml_sessions_controller.rb

```diff
@@ -5,10 +5,10 @@ class Devise::SamlSessionsController < Devise::SessionsController
   unloadable if Rails::VERSION::MAJOR < 4
   if Rails::VERSION::MAJOR < 5
     skip_before_filter :verify_authenticity_token
-    prepend_before_filter :store_info_for_sp_initiated_logout, only: :destroy
+    prepend_before_filter :verify_authenticity_token, :store_info_for_sp_initiated_logout, only: :destroy
```

Comment:
Should this be the same as in the below clause?

```suggestion
    prepend_before_filter :verify_signed_out_user, :store_info_for_sp_initiated_logout, only: :destroy
```

Reply:
@adamstegman Thank for your comment. I've already fixed it and added rspec.
Could you take a look :eyes: